backend/src/main/kotlin/com/pk/zbtz/zbtzbackend/controllers/movie/service/PostgresMovieService.kt

```diff
@@ -0,0 +1,130 @@
+package com.pk.zbtz.zbtzbackend.com.pk.zbtz.zbtzbackend.controllers.movie.service
+
+import com.pk.zbtz.zbtzbackend.controllers.ResponseWithStatistics
+import com.pk.zbtz.zbtzbackend.controllers.movie.requests_and_responses.AddMovieRequest
+import com.pk.zbtz.zbtzbackend.controllers.movie.requests_and_responses.GetMoviesResponse
+import com.pk.zbtz.zbtzbackend.controllers.movie.requests_and_responses.GetMoviesSorting
+import com.pk.zbtz.zbtzbackend.controllers.movie.requests_and_responses.GetMoviesSortingOrder
+import com.pk.zbtz.zbtzbackend.controllers.movie.service.MovieService
+import com.pk.zbtz.zbtzbackend.databases.postgres.models.MoviePostgresModel
+import com.pk.zbtz.zbtzbackend.domain.Movie
+import com.pk.zbtz.zbtzbackend.domain.MovieSummary
+import com.pk.zbtz.zbtzbackend.domain.Statistics
+import com.pk.zbtz.zbtzbackend.utils.execution_timer.ExecutionTimer
+import org.springframework.stereotype.Service
+import kotlin.math.ceil
+
+@Service
+class PostgresMovieService(
+    private val executionTimer: ExecutionTimer,
+    private val service: com.pk.zbtz.zbtzbackend.com.pk.zbtz.zbtzbackend.databases.postgres.services.MovieService
```

Comment:
```suggestion
    private val service: MovieService
```

Reply:
k

Reply:
not possible, naming clash will not change